integration_tests/e2e/goal/createGoal.cy.ts

```diff
@@ -29,18 +30,24 @@ context('Create a goal', () => {
     cy.signIn()
     cy.visit(`/plan/${prisonNumber}/goals/create`)
 
-    const page = Page.verifyOnPage(CreateGoalPage)
-    page.setGoalTitle('Learn French')
-    page.setGoalReviewDate(23, 12, 2024)
-
-    // When
-    page.submitPage()
+    const createGoal = Page.verifyOnPage(CreateGoalPage)
+    createGoal.setGoalTitle('Learn French')
+    createGoal.setGoalReviewDate(23, 12, 2024)
+    createGoal.submitPage()
 
-    // Then
     const addStepPage = Page.verifyOnPage(AddStepPage)
     addStepPage.isForGoal('Learn French')
 
     addStepPage.setStepTitle('Book French course')
     addStepPage.setStepTargetDate(23, 12, 2024)
+
+    // When
+    addStepPage.submitPage()
+
+    // Then
+    const addNotePage = Page.verifyOnPage(AddNotePage)
+    addNotePage.isForPrisoner(prisonNumber)
```

Comment:
I like where this is going as an E2E test; but we are losing some of the intermediate page assertions that we had previously. IE. we've lost the assertions of `isForPrisoner` on the CreateGoal page and Add Note page

I'm not sure I want to add assertions like that to _this_ test, because otherwise it will become massive (and the test is about the E2E journey, not about whether interim pages are rendered correctly ..... Hmmmm, we'll need to give this some thought .... 🤔